keras_core/backend/torch/numpy.py

```diff
@@ -44,7 +44,7 @@ def mean(x, axis=None, keepdims=False):
     if isinstance(x, (list, tuple)):
         x = stack(x)
     x = convert_to_tensor(x)
-    if axis == () or axis == []:
+    if axis == None or axis == () or axis == []:
```

Comment:
`axis=None` means "reduce all dimensions" so returning `x` would be the wrong behavior IMO